client/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -54,43 +93,66 @@ tie.factory('PythonCodePreprocessorService', [
       correctnessTests.forEach(function(test, index) {
         testCode += '\n' + [
           VARNAME_TEST_RESULTS + '.append(',
-          '    run_test(' + jsonVariableToPython(test.getInput()) + '))'
+          '    run_correctness_test(' + jsonVariableToPython(test.getInput()) + '))'
         ].join('\n');
       });
 
       return testCode;
-    }
+    };
 
-    return {
-      // Wraps a WRAPPER_CLASS_NAME class around the series of functions in a
-      // given code snippet.
-      _wrapCodeIntoClass: function(code) {
-        var codeLines = code.trim().split('\n');
+    var generatePerformanceTestCode = function(performanceTests) {
+      performanceTests.forEach(function(test, index) {
+        var qualifiedEvaluationFunctionName = (
+          WRAPPER_CLASS_NAME + '().' + test.getEvaluationFunction());
+        var qualifiedTransformationFunctionName = (
+          WRAPPER_CLASS_NAME + '().' + test.getTransformationFunction());
+        // TODO(eyurko): Make this work for other runtimes.
+        // TODO(eyurko): 30 seems very scientific. Maybe stop hardcoding?
```

Comment:
Also, for now, at least give the three constants names and pull them out. You could do something like this, for example:

```
    SMALL_INPUT_SIZE = 10
    LARGE_INPUT_SIZE = 100
    UPPER_BOUND_RATIO_IF_LINEAR = (LARGE_INPUT_SIZE / SMALL_INPUT_SIZE) * 3
```

(feel free to pick better descriptions, but at least give some indication of what is going on).